src/rbdmap

```diff
@@ -1,15 +1,6 @@
 #!/bin/bash
 
 do_map() {
-
-        # default to reasonable value if RBDMAPFILE not set in environment
-        printenv RBDMAPFILE >/dev/null || local RBDMAPFILE=/etc/ceph/rbdmap
-
-	if [ ! -f "$RBDMAPFILE" ]; then
-		logger -p "daemon.warning" -t rbdmap "No $RBDMAPFILE found."
-		exit 0
-	fi
-
```

Comment:
There is a teuthology test that might be affected by this: https://github.com/ceph/ceph/blob/master/qa/suites/rbd/singleton/all/rbdmap_RBDMAPFILE.yaml